core/store/modules/attribute/types/AttributeState.ts

```diff
@@ -0,0 +1,5 @@
+export default interface AttributeState {
```

Comment:
Shouldn;t it be just 'Attribute' ?

Reply:
I think it should stay AttributeState as it's vuex state object

Reply:
But it's also the same object as the one in View